templates/test/live.html.ep

```diff
@@ -1,24 +1,92 @@
+            % else {
+                no developer session opended
+                % if (is_admin) {
+                    - click to open
+                % }
+            % }
+        </div>
     </div>
-    <div id="running_module" class='col-sm-9'>Unknown</div>
-  </div>
-  % if (my $developer_session = $job->developer_session) {
-    <div class='row'>
-        <div class='col-sm-3'>
-        Developer session:
+    <div class="card-body">
+        <form action="#" method="get" id="developer-form" class="developer-mode-element" data-hidden-on="lockedByOtherDeveloper">
+            <p class="developer-mode-element" data-hidden-on="ownSession">
+                Select what you want to do and confirm using the button below. This will start a developer
+                session which will lock those developer options for other users.
+            </p>
+            <p class="developer-mode-element" data-visible-on="ownSession">
+                You started a developer session. Use the controls above to control the test behavior.
+            </p>
+            <h3>
+                Pause test execution
+                <%= help_popover('Help for <i>pause test execution</i>' => '
+                        <p>Pauses the test execution. So far it is only possible to pause at a
+                        specific module. The test can then be accessed via VNC for further
+                        investigation.</p>
+                        <p>Resuming the job after it has been paused is possible.</p>')
+                %>
+            </h3>
+            <div class="form-group row">
+                <label for="developer-pause-at-module" class="col-sm-3 col-form-label">
+                    at specific module
+                </label>
+                <div class="col-sm-9">
+                    %= include 'test/module_select'
+                </div>
+            </div>
+            <hr>
+            <div>
+                <a href="#" onclick="startDeveloperSession(); return false;"
+                    class="btn btn-warning developer-mode-element" data-hidden-on="ownSession">
+                    <i class="fas fa-lock"></i> Confirm & start developer session
+                </a>
+                <a href="#" onclick="quitDeveloperSession(); return false;"
+                    class="btn btn-danger developer-mode-element" data-visible-on="ownSession">
+                    <i class="far fa-stop-circle"></i> Cancel job
+                </a>
+                <a href="#" onclick="resumeTestExecution(); return false;" class="btn btn-secondary developer-mode-element" data-visible-on="isPaused">
+                    <i class="far fa-play-circle"></i> Resume test execution
+                </a>
+                <a href="<%= url_for('developer_ws_console')->query({proxy => 1}) %>" target="blank"
+                    class="btn btn-secondary developer-mode-element" data-visible-on="ownSession">
+                    <i class="fas fa-terminal"></i> Open console
```

Comment:
Hmmm I think it's safe to hide the console.

Reply:
Yes, I'll get rid of the console button. Or maybe I show it only when launching the Mojo app in development mode? Because when developing (the developer mode) it can still be useful to open that console.

Reply:
implemented